PWGHF/TableProducer/candidateSelectorDplusToPiKPi.cxx

```diff
@@ -45,6 +45,42 @@ struct HfCandidateSelectorDplusToPiKPi {
   // topological cuts
   Configurable<std::vector<double>> binsPt{"binsPt", std::vector<double>{hf_cuts_dplus_to_pi_k_pi::vecBinsPt}, "pT bin limits"};
   Configurable<LabeledArray<double>> cuts{"cuts", {hf_cuts_dplus_to_pi_k_pi::cuts[0], nBinsPt, nCutVars, labelsPt, labelsCutVar}, "Dplus candidate selection per pT bin"};
+  // QA switch
+  Configurable<bool> activateQA{"activateQA", false, "Flag to enable QA histogram"};
+
+  TrackSelectorPID selectorPion;
+  TrackSelectorPID selectorKaon;
+
+  HistogramRegistry registry{"registry"};
+
+  void init(InitContext const&)
+  {
+    selectorPion.setPDG(kPiPlus);
+    selectorPion.setRangePtTPC(ptPidTpcMin, ptPidTpcMax);
+    selectorPion.setRangeNSigmaTPC(-nSigmaTpcMax, nSigmaTpcMax);
+    selectorPion.setRangePtTOF(ptPidTofMin, ptPidTofMax);
+    selectorPion.setRangeNSigmaTOF(-nSigmaTofMax, nSigmaTofMax);
+
+    selectorKaon.setPDG(kKPlus);
+    selectorKaon.setRangePtTPC(ptPidTpcMin, ptPidTpcMax);
+    selectorKaon.setRangeNSigmaTPC(-nSigmaTpcMax, nSigmaTpcMax);
+    selectorKaon.setRangePtTOF(ptPidTofMin, ptPidTofMax);
+    selectorKaon.setRangeNSigmaTOF(-nSigmaTofMax, nSigmaTofMax);
```

Comment:
Can this not be replaced by just copying from `selectorPion` as:
```c++
selectorKaon = selectorPion;
selectorKaon.setPDG(kKPlus);
```
?

Reply:
Yes, it does :)